build_variables.bzl

```diff
@@ -895,6 +895,7 @@ libtorch_python_core_sources = [
     "torch/csrc/fx/node.cpp",
     "torch/csrc/mps/Module.cpp",
     "torch/csrc/mtia/Module.cpp",
+    "torch/csrc/python_custom_backend/Module.cpp",
```

Comment:
The name `PrivateUse1` is not very formal, we need a name related to a general accelerator, similar to cpu, cuda, etc., so I think `acc` may be a suitable name

Reply:
done.